### ISSUE TITLE
3++ Publishing (To Web and Onto server)

### ISSUE THREAD

**Issue by GSChandel** [NHSDigital/DataDictionaryPublication#243]
Capability to publish the Data Dictionary to the Web is required. Amendments to the current process developed for 3+ are required to make publishing ‘single click’ for 3++.  

The publishing process takes data from the Mauro Data Dictionary, blends with additional textural information and publishes to the Web. 

Publishing to a Pre-Live and Live website required. 

Publishing to also ‘push’ appropriate data to the Onto Server.

**Comment by maxChappell** [comment]
This requirement is in development by OCC now - MS 29, MS26 - May be a gap re the web publication itself (for a release) - not explicitly mentioned in the milestones.

**Comment by GSChandel** [comment]
JIRA Ticket# [DPTC-2805](https://nhsd-jira.digital.nhs.uk/browse/DPTC-2805)

**Comment by GSChandel** [comment]
@rajabr58 to work with Kun to work on the required Epic for both #243 and #305

**Comment by AngelaFaulding** [comment]
@rajabr58 - this includes Jira tickets:
- To generate the DD pipeline for the archive version of the dictionary: https://nhsd-jira.digital.nhs.uk/browse/DPNDA-287
- To generate the DD pipeline for Change Request previews of the dictionary: https://nhsd-jira.digital.nhs.uk/browse/DPNDA-291.

Kun is also working on "To generate the DD pipeline for Change Request pdf": https://nhsd-jira.digital.nhs.uk/browse/DPNDA-322.
This is part of the publishing pipeline but the output does not go to the cloud.

**Comment by rajabr58** [comment]
Still in progress - ALB NLB issue has been unblocked, testing today/tomorrow with a view to going live next week.

**Comment by AngelaFaulding** [comment]
8/2/22 - checked with Rachel what this means.
The JIRA ticket is "Web Migration of Data Dictionary to AWS".
Unfortunately they are still having Network Load Balancer issues. So they have had to go back to AWS for support.

**Comment by rajabr58** [comment]
NLB/ALB now working.  Kun to contact DNS team to update IP

**Comment by rajabr58** [comment]
https://nhsd-jira.digital.nhs.uk/browse/DPTC-2805

**Comment by GSChandel** [comment]
> @rajabr58 - this includes Jira tickets:
> 
> * To generate the DD pipeline for the archive version of the dictionary: https://nhsd-jira.digital.nhs.uk/browse/DPNDA-287
> * To generate the DD pipeline for Change Request previews of the dictionary: https://nhsd-jira.digital.nhs.uk/browse/DPNDA-291.
> 
> Kun is also working on "To generate the DD pipeline for Change Request pdf": https://nhsd-jira.digital.nhs.uk/browse/DPNDA-322. This is part of the publishing pipeline but the output does not go to the cloud.

The JIRA Ticket [DPNDA-287](https://nhsd-jira.digital.nhs.uk/browse/DPNDA-287) is now replaced with a new ticket - [DPNDA-290](https://nhsd-jira.digital.nhs.uk/browse/DPNDA-290)

**Comment by AngelaFaulding** [comment]
Ticket has been closed as the DD is now published to the Cloud.
Replaced with ticket: #406
